index.bs

```diff
@@ -304,12 +304,10 @@ enum XREnvironmentBlendMode {
   readonly attribute XRSessionMode mode;
   readonly attribute XRPresentationContext? outputContext;
   readonly attribute XREnvironmentBlendMode environmentBlendMode;
-
-  attribute double depthNear;
-  attribute double depthFar;
-  attribute XRLayer? baseLayer;
+  readonly attribute XRRenderState renderState;
 
   // Methods
+  Promise&lt;void&gt; updateRenderState(optional XRRenderStateInit state);
```

Comment:
Not super critical for this change but we should have spec language that states when this function throws an exception on bad input. For instance, if the baseLayer passed in does not belong to the current session, that should throw an exception.